xchange-cexio/src/main/java/info/bitrich/xchangestream/cexio/CexioAdapters.java

```diff
@@ -71,4 +78,53 @@ private static BigDecimal adaptAmount(BigDecimal amount) {
         }
         return status;
     }
+    
+    private static List<LimitOrder> updateOrInsertQuantityForPrice(List<LimitOrder> existingOrders, 
```

Comment:
Apologies if I've not read this in enough detail, but are you aware of all the `update` methods on `OrderBook` which are intended to do this sort of thing?  Is this custom logic definitely necessary?

In some cases it is; some exchanges are weird.  Just want to check.

Reply:
OrderBook update did exactly what I needed. This is fixed in d1c7ab91daaa8867f279993a72e812c2e3dd4e90